src/mcdu-server/index.js

```diff
@@ -0,0 +1,251 @@
+/* eslint-disable no-console */
+
+'use strict';
+
+const WebSocket = require('ws');
+const http = require('http');
+const path = require('path');
+const fs = require('fs');
+const network = require('network');
+const print = require('pdf-to-printer');
+const os = require('os');
+const PDFDocument = require('pdfkit');
+require('./standalone-patch');
+
+// This tells pkg to include these files in the binary
+path.join(__dirname, 'client/build/android-chrome-192x192.png');
+path.join(__dirname, 'client/build/android-chrome-512x512.png');
+path.join(__dirname, 'client/build/apple-touch-icon-180x180.png');
+path.join(__dirname, 'client/build/bundle.js');
+path.join(__dirname, 'client/build/favicon.ico');
+path.join(__dirname, 'client/build/HoneywellMCDU.ttf');
+path.join(__dirname, 'client/build/HoneywellMCDUSmall.ttf');
+path.join(__dirname, 'client/build/index.html');
+path.join(__dirname, 'client/build/mcdu-r2-c.png');
+path.join(__dirname, '../../node_modules/pdf-to-printer/dist/SumatraPDF.exe');
+path.join(__dirname, '../../node_modules/linebreak/src/classes.trie');
+path.join(__dirname, '../../node_modules/pdfkit/js/data/Helvetica.afm');
+
+const sumatraPdfPath = path.join(os.tmpdir(), 'SumatraPDF.exe');
+fs.copyFileSync(path.join(__dirname, '../../node_modules/pdf-to-printer/dist/SumatraPDF.exe'), sumatraPdfPath);
```

Comment:
Can you explain this part a little?

Reply:
The library that I'm using to to print, _pdf-to-printer_ uses an open source program called _SumatraPDF_ to communicate with the printer driver. This code extracts `SumatraPDF.exe` to a temporary folder so it can be run by _pdf-to-printer_.

Reply:
So it can't run it from the node_modules directory?

Reply:
It can't run from _pkg_'s virtual filesystem when it's packaged in an exe. Windows needs it to be its own file.

Reply:
Ah, that makes sense!